src/dtx/dtx_srv.c

```diff
@@ -92,7 +94,31 @@ dtx_handler(crt_rpc_t *rpc)
 		else
 			rc = vos_dtx_check(cont->sc_hdl,
 					   din->di_dtx_array.ca_arrays,
-					   NULL, NULL, false);
+					   NULL, NULL, NULL, false);
+		break;
+	case DTX_REFRESH:
```

Comment:
Could you add some comments to explain "DTX_REFRESH"?

Reply:
Because of async batched commit semantics, the DTX status on the leader maybe different from the one on non-leaders. For the leader, it exactly knows whether the DTX is committable or not, but the non-leader does not know if the DTX is in 'prepared' status. So someone needs to know whether the 'prepared' DTX is real committable or not, the non-leader need to refresh its DTX status from the leader. The DTX_REFRESH RPC is used for that. I will add comment in the patch.